web/impact/impact/v1/helpers/model_helper.py

```diff
@@ -17,6 +37,97 @@ def field_value(self, field):
             return result
         return getattr(self.subject, field, None)
 
+    def field_setter(self, field, value):
```

Comment:
Certainly not a blocker, but when I saw the name `field_setter` I guessed that a callable would be returned. Something like `set_field` seems like it would be more straightforward for this.